Tests/Unit/Mailer/Transport/SparkpostTransportTest.php

```diff
@@ -28,17 +28,17 @@
 
 class SparkpostTransportTest extends TestCase
 {
-    private TranslatorInterface|MockObject $translatorMock;
+    private TranslatorInterface&MockObject $translatorMock;
```

Comment:
Intersection types are only allowed since PHP 8.1. Since, we are supporting PHP 8.0 as well, I think we should not make this change.

Reply:
Good point!